### PR TITLE
Check the combined status before updating

### PR DESCRIPTION
**Pull request by jakirkham**
Alternative to PR ( https://github.com/conda-forge/conda-forge-webservices/pull/235 ).

Only updates the webservice if the combined status is a pass. This should avoid multiple updates to the webservice when the package is not there. Though it may mean the specific architecture and Python version that we need has already been deployed for a little bit.

cc @isuruf

### CODE REVIEW

File: conda_forge_webservices/webapp.py [hunk 1]
```diff
@@ -268,10 +279,25 @@ def post(self):
...
+                token = os.environ.get('GH_TOKEN')
+                state = get_combined_status(token, repo_name, sha)
+
+            # Update if the combined status is a success
+            if state == 'success':
+                update_me.update_me()
+
             print_rate_limiting_info()
-        else:
+        elif event != 'push':
```
Comment (jakirkham) [inline conda_forge_webservices/webapp.py]: Maybe this should just be disabled on the webhooks. Thoughts?

**Comment by isuruf** [comment]
The way I've written the webhook is that it checks the versions of the conda packages in its environment against the latest versions available in anaconda.org. So, it doesn't matter whether we check the combined status or just any status.

**Comment by jakirkham** [comment]
Not sure I follow. If one looks at the commits on `master` recently, one can see there are a bunch of commits to update `conda-build`. It would be better to have just one. That's what this PR is solving.

File: conda_forge_webservices/webapp.py (hunk 1 above)
Comment (scopatz) [reply conda_forge_webservices/webapp.py]: Seems fine.

**Comment by jakirkham** [comment]
Thoughts on this @conda-forge/core?

Recently (past 30 mins) we had 15 commits pushed to webservices to update the `conda-build` version. This should reduce it to 1.

**Comment by isuruf** [comment]
See my comment.

> So, it doesn't matter whether we check the combined status or just any status.

Having the 15 commits some other issue.

**Pull request merged**